### PR TITLE
GUI-2863 use separate file for oidc credentials

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-2863

### CODE REVIEW

**Comment by dkavanagh** [comment]
hal, retest this please

File: conf/console.default.ini [hunk 1]
```diff
@@ -21,10 +21,8 @@ ufsport = 8773
 # Console must be using https and registration process with ident provider is required.
 # ident host
 #oidc.hostname = 
-# client id for this installation (needs to come from ident provider)
-oidc.client.id = 
-# client secret for this installation (needs to come from ident provider)
-oidc.client.secret = 
+# This file contains client id and secret that are generated by the OIDC provider
+oidc.client.ini = /etc/eucaconsole/oidc-credentials.ini
```
Comment (kamalgill) [inline conf/console.default.ini]: Do we need a DOC ticket to add this setting to our documentation?

Comment (dkavanagh) [inline conf/console.default.ini]: Definitely. @kerimeredith do you need me to write that?

**Comment by kerimeredith** [comment]
@dkavanagh yes please describe what needs to change in the ini file and any other related doc change required. Also confirm it is 4.4 work.

**Pull request merged**